src/Components/Dashboard/SendNFT/sendNft.js

```diff
@@ -340,7 +340,10 @@ const SendNft = () => {
           </div>
         </Modal.Header>
         <Modal.Body>
-          <div className={styles.modal__body__wrapper}>
+          <div
+            style={{ height: "80vh", overflow: "auto" }}
```

Comment:
Please try not use the inline css, it may affect the mobile version. Please add the css in css file